data/src/main/java/ca/bc/gov/data/local/entity/VaccineRecordEntity.kt

```diff
@@ -34,6 +34,6 @@ data class VaccineRecordEntity(
     val shcUri: String,
     @ColumnInfo(name = "federal_pass")
     val federalPass: String?,
-    @ColumnInfo(name = "data_source")
-    val dataSource: DataSource
+    @ColumnInfo(name = "data_source", defaultValue = "")
```

Comment:
@ColumnInfo(name = "data_source")
    val dataSource: DataSource = Datasource.PUBLIC_API
 should suffice the default value requirement as this columns already exists and is having data.